src/index.html

```diff
@@ -5,9 +5,47 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <div class="card__product">
+        <div class="card__images">
+          <img
+              src="./images/imac.jpeg"
+              alt="images-product"
+          >
```

Comment:
Add classes for this element
```suggestion
          <img
            class="card__image"
            src="./images/imac.jpeg"
            alt="images-product"
          >
```